docs/components/DataList/DataList.stories.mdx

```diff
@@ -23,3 +25,201 @@ item.
+
+<ArgsTable of={DataList} />
+
+## Layout
+
+DataList.Layout enables you to implement the layout of the list items. Keep in
+mind that DataList doesn't have a default layout so you have to have at least 1
+implemented.
+
+It also supports different `size` breakpoints so you can completely change the
+component it's using depending on the window size.
+
+It is also important to add the type of `DataListItem<typeof data>` on the
+`item` param in the layout so you can easily access the known data.
+
+```tsx
+// You can have 1 responsive layout
+<DataList>
+  <DataList.Layout>
+    {/* Declare the type of the item */}
+    {(item: DataListItem<MyDataType>) => (
+      <Grid>
+        <Grid.Cell size={{ xs: 12 md: 6 }}>{item.name}</Grid.Cell>
+        <Grid.Cell size={{ xs: 12 md: 6 }}>{item.email}</Grid.Cell>
+      </Grid>
+    )}
+  </DataList.Layout>
+</DataList>
+
+// Or you can have multiple layouts per breakpoint
+<DataList>
+  <DataList.Layout size="xs">
+    {(item: DataListItem<MyDataType>) => (
+      <Grid>
+        <Grid.Cell size={{ md: 6 }}>{item.name}</Grid.Cell>
+        <Grid.Cell size={{ md: 6 }}>{item.email}</Grid.Cell>
+      </Grid>
+    )}
+  </DataList.Layout>
+
+  <DataList.Layout size="md">
+    {(item: DataListItem<MyDataType>) => (
+      <Content>
+        <div>{item.name}</div>
+        <div>{item.email}</div>
+      </Content>
+    )}
+  </DataList.Layout>
+</DataList>
+```
+
+<ArgsTable of={DataList.Layout} />
+
+## Per item action(s)
+
+DataList.ItemActions allows you to add an action on the item itself such as
+`onClick` or `url`. While the DataList.ItemAction (without an s) child adds more
+action that is related to the item.
+
+```tsx
+<DataList>
+  <DataList.ItemActions onClick={item => alert(item.name)}>
+    <DataList.ItemAction icon="edit" label="Edit" onClick={} />
+    <DataList.ItemAction icon="delete" label="Delete" onClick={} />
+  </DataList.ItemActions>
+</DataList>
+```
+
+<Tabs>
+  <Tab label="DataList.ItemActions">
+    <ArgsTable of={DataList.ItemActions} />
+  </Tab>
+  <Tab label="DataList.ItemAction">
+    <ArgsTable of={DataList.ItemAction} />
+  </Tab>
+</Tabs>
+
+## Layout actions
+
+By default, the actions from the DataList.ItemActions will be placed on the top
+right of the list item and will be triggered by hover. You can change this
+behaviour by including the DataList.LayoutActions component inside the
+DataList.Layout component.
+
+```tsx
+<DataList>
+  <DataList.Layout>
+    {(item: DataListItem<MyDataType>) => (
+      <>
+        <Grid>
+          <Grid.Cell size={{ xs: 12 md: 6 }}>{item.name}</Grid.Cell>
+          <Grid.Cell size={{ xs: 12 md: 6 }}>{item.email}</Grid.Cell>
+        </Grid>
+
+        {/* This will place the actions as a part of the layout */}
+        <DataList.LayoutActions />
+      </>
+    )}
+  </DataList.Layout>
+</DataList>
+```
+
+## Filters
+
+DataList.Filters add filters to the list. This compound component accepts any
```

Comment:
```suggestion
DataList.Filters adds filters to the list. This compound component accepts any
```

Reply:
grammarly says no? 

<img width="922" alt="image" src="https://github.com/GetJobber/atlantis/assets/15986172/bcadc82b-076e-4b4e-8d3b-80ae69fee521">

Reply:
Huh I guess that's a weird one...

In the sense that `DataList.Filters` is _one compound component_ that allows you to add filters, then it should be "adds".

In the sense that `Filters` is plural, and you are able to add _multiple filters_, then it should be "add".

Reply:
you're probably right. grammarly gets confused with new lines and code as english a lot haha

Reply:
anyway, it's adds now